MonoGame.Framework/Graphics/Shader/Shader.OpenGL.cs

```diff
@@ -128,11 +128,14 @@ protected override void Dispose(bool disposing)
             if (!IsDisposed && _shaderHandle != -1)
             {
                 Threading.BlockOnUIThread(() =>
+                {
+                    if (!IsDisposed)
```

Comment:
Ow, I added this because the GL.DeleteShader call caused a crash sometimes because it was being called twice and I thought the dispose action might have been queued twice, but that is not the case since IsDisposed will return true after the base.Dispose call. I accidentally added this somewhere in a commit somewhere, but will revert this. Details of how and when the crash happened are here: 
https://github.com/MonoGame/MonoGame/pull/5429/commits/8fd62bb39838d33e59d45982bff40f07d883b88d#diff-d60307831e093a2b2bddf43a3ad0dd95R20